From e2c002d6dbf9364f682ee103521a3562ebc02ef2 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Steven!=20Ragnar=C3=B6k?= <steven@nuclearsandwich.com>
Date: Wed, 17 Jun 2020 02:22:15 +0000
Subject: [PATCH 2/2] Update patches to depend on rolling instead of foxy.

---
 debian/control.em | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/control.em b/debian/control.em
index 75f8825..b82e69a 100644
--- a/debian/control.em
+++ b/debian/control.em
@@ -8,7 +8,7 @@ Standards-Version: 3.9.2
 
 Package: @(Package)
 Architecture: any
-Depends: ${shlibs:Depends}, ${misc:Depends}, @(', '.join(Depends)), ros-foxy-rmw-fastrtps-cpp | ros-foxy-rmw-connext-cpp
+Depends: ${shlibs:Depends}, ${misc:Depends}, @(', '.join(Depends)), ros-rolling-rmw-fastrtps-cpp | ros-rolling-rmw-connext-cpp | ros-rolling-rmw-cyclonedds-cpp
 @[if Conflicts]Conflicts: @(', '.join(Conflicts))@\n@[end if]@
 @[if Replaces]Replaces: @(', '.join(Replaces))@\n@[end if]@
 Description: @(Description)
-- 
2.27.0

